src/containers/profile/timeline.svelte

```diff
@@ -1,49 +1,80 @@
 <script>
-  import { createEventDispatcher } from 'svelte';
-  import Button from 'ui/button.svelte';
+  import { createEventDispatcher, getContext } from 'svelte';
+  import { Button, Loading } from 'attractions';
+  import { snackbarContextKey } from 'attractions/snackbar';
+  import EmptyState from '@/components/common/empty-state.svelte';
   import TimelineEntry from '@/components/profile/timeline-entry.svelte';
+  import LeaveFeedbackModal from '@/components/projects/view/leave-feedback-modal.svelte';
+  import * as api from '@/utils/api.js';
 
   export let timelinePromises;
-
-  const dispatch = createEventDispatcher();
+  export let competences;
 
   function requestMore() {
     dispatch('more-timeline');
   }
```

Comment:
You usually inline such one-line functions. It is only used once

Reply:
This was there from a long time ago. I think I did it this way back then to emphasize what the function is doing.